jplag/src/test/java/de/jplag/special/TokenPrinterTest.java

```diff
@@ -7,30 +7,31 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-import de.jplag.JPlagResult;
-import de.jplag.Submission;
-import de.jplag.TestBase;
-import de.jplag.TokenPrinter;
+import de.jplag.*;
 import de.jplag.exceptions.ExitException;
 import de.jplag.options.JPlagOptions;
-import de.jplag.options.LanguageOption;
 
 /**
  * Special test case the does not really test anything but prints the tokens and the corresponding line of code.
  * @author Timur Saglam
  */
+// TODO DF: We could move some tests to their language frontends.
```

Comment:
This is not a test per se, this is more like a debug utility tool. We could create an abstract token printer test in the test-utils and then add subclasses to each frontend. But I am not sure if that is so much better.